src/app/common/transactions/stacks/transaction.utils.ts

```diff
@@ -126,3 +127,16 @@ export function getTxSenderAddress(tx: StacksTransaction): string | undefined {
   );
   return txSender;
 }
+
+export function getEstimatedConfirmationTime(
+  isTestnet: boolean,
```

Comment:
Better to avoid booleans to describe network, as it can be > 2

Why not pass `network` string and use `whenNetwork`?

Reply:
I don't see that the `NetworkMode` type used in `whenNetwork` is exposed anywhere for Stacks? I only see it used with Bitcoin. Our network types are pretty confusing. `network` only exists on `BitcoinChainConfig` not `StacksChainConfig`.